test/hotspot/jtreg/gc/arguments/TestParallelGCThreads.java

```diff
@@ -87,14 +88,17 @@ public static void testFlags() throws Exception {
     if (GC.Parallel.isSupported()) {
       supportedGC.add("Parallel");
     }
+    if (GC.Z.isSupported()) {
+      supportedGC.add("Z");
+    }
```

Comment:
The previous comment, show below, needs to be adjusted as well.

```
// For each parallel collector (G1, Parallel)
```

And should we also add `Shenandoah` here?